### ISSUE TITLE
Support * anywhere in column specifications

### ISSUE THREAD

**Issue by scudette** [Velocidex/velociraptor#725]
Currently we can do something like
```
SELECT *, Foo, Bar FROM...
```
But this wont work
```
SELECT Foo, Bar, * FROM ...
```

It would be nice to support that or even:
```
SELECT Foo, *, Bar FROM ...
```

**Comment by scudette** [comment]
This is probably best solved with the plugin solution described in #969

**Comment by scudette** [comment]
It looks like mysql has similar semantics

https://stackoverflow.com/questions/3999491/how-do-i-select-all-the-columns-from-a-table-plus-additional-columns-like-rownu
